test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
We can remove this file to use the Makefile instead (the Makefile will just need to be updated to use `SQLITE_DATASOURCE`